mobile/src/features/Discover/useCases/SelectDappFromList/SelectDappFromListScreen.tsx

```diff
@@ -40,6 +41,8 @@ export const SelectDappFromListScreen = () => {
   )
   const [isShowedWelcomeDApp] = useShowWelcomeDApp()
 
+  usePageViewTracking('Discover Page Viewed')
```

Comment:
for page view tracking.
@banklesss just look at the _layout example in the ctrl repo. you just need to subscribe to the global navigation listener and it will let you track every screen, rather than adding this into every screen. Please remove it from every screen.